### ISSUE TITLE
Stick nav bar on blog and docs

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#4761]
If someone goes to opencollective.com and then goes to our docs or blog, they loose the top nav bar and the whole context of where they are and what else to look at. It's pretty hard to get from docs and blog to the rest of the site if you start there, or back to the homepage if you came from there. We should have a nav bar that is there all the time (and probably blog should be in it as well).

**Comment by SudharakaP** [comment]
I agree. Myself find scrolling up and down all the time to get to different sections. I am thinking if we could just stick the current nav bar as a quick fix for this. 🤔

**Comment by Betree** [comment]
It's not that simple: our docs live on a different provider/domain (docs.opencollective.com). To keep the top bar fixed, we would have to render the docs in an embed iframe, which has the significant downside of not preserving the URL.

I see two solutions here:
1. We go with this solution of using an iframe. We should then make sure we keep a unique URL for each documentation page (e.g. `https://opencollective.com/docs/settings/customization` => will render `https://docs.opencollective.com/settings/customization`).
2. We add custom links in Gitbook to quickly go back to Open Collective. I just enabled that as it's a more straightforward solution from a design/engineering perspective, but that might not be enough

![image](https://user-images.githubusercontent.com/1556356/135810051-2813c0e1-49c6-4388-b084-bff0695cd1ce.png)
